src/System.IO.Pipes/src/System/IO/Pipes/AnonymousPipeServerStreamAcl.cs

```diff
@@ -0,0 +1,25 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.IO.Pipes
+{
+    public static class AnonymousPipeServerStreamAcl
+    {
+        /// <summary>
+        /// Creates a new instance of the <see cref="AnonymousPipeServerStream" /> class with the specified pipe direction, inheritability mode, buffer size, and pipe security.
+        /// </summary>
+        /// <param name="direction">One of the enumeration values that determines the direction of the pipe. Anonymous pipes can only be in one direction, so direction cannot be set to <see cref="PipeDirection.InOut" />.</param>
+        /// <param name="inheritability">One of the enumeration values that determines whether the underlying handle can be inherited by child processes.</param>
+        /// <param name="bufferSize">The size of the buffer. This value must be greater than or equal to 0.</param>
+        /// <param name="pipeSecurity">An object that determines the access control and audit security for the pipe.</param>
+        /// <returns>A new anonymous pipe server stream instance.</returns>
+        /// <remarks>Setting <paramref name="pipeSecurity" /> to <see langword="null" /> is equivalent to calling the `System.IO.Pipes.AnonymousPipeServerStream(System.IO.Pipes.PipeDirection direction, System.IO.HandleInheritability inheritability, int bufferSize)` constructor directly.</remarks>
+        /// <exception cref="NotSupportedException"><paramref name="direction" /> is <see cref="PipeDirection.InOut" />.</exception>
+        /// <exception cref="ArgumentOutOfRangeException"><paramref name="inheritability" /> is not set to a valid <see cref="HandleInheritability" /> enum value.</exception>
+        public static AnonymousPipeServerStream Create(PipeDirection direction, HandleInheritability inheritability, int bufferSize, PipeSecurity pipeSecurity)
```

Comment:
I'm not understanding the design.  If we're adding public API based on PipeSecurity to the System.IO.Pipes library, why are we adding this new method rather than exposing the same ctor:
```C#
public AnonymousPipeServerStream(PipeDirection direction, HandleInheritability inheritability, int bufferSize, PipeSecurity pipeSecurity)
```
that already exists in .NET Framework?

Reply:
Your comment makes sense.
I'm not sure what the right thing to do would be, since the API is already approved as is.
Should we cancel this PR and instead submit a new API approval to make the existing constructor public? @JeremyKuhne

Reply:
It is only exposed in the `System.IO.Pipes.AccessControl` ref assembly.